image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -165,7 +165,7 @@ defaults:
 
 [<- if .FeatureFlags.ROX_SCANNER_V4_SUPPORT >]
   scannerV4:
-    disable: true
```

Comment:
How does this change affect operator-based installations?

Reply:
For operator-based installations the same strategy is implemented (@johannes94, please correct me if wrong). The expectation from the perspective of the Scanner V4 initiative is that Scanner V4 will be pulled in automatically, unless disabled explicitly (which won't be a supported configuration).

Reply:
Ah, ok, I do see `scanner-v4` in the events [logged in the operator test job](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/stackrox_stackrox/9529/pull-ci-stackrox-stackrox-master-ocp-4-11-operator-e2e-tests/1749926725652844544/artifacts/operator-e2e-tests/stackrox-stackrox-e2e-test/build-log.txt). So this change turns it on by default for both types.

I guess we'll want some assertions added to the operator e2e tests to make sure it's healthy and all, but I guess this can be done in a separate PR.